src/app/builder/builder.component.ts

```diff
@@ -21,6 +21,7 @@ export class BuilderComponent implements OnInit {
   props: BuilderProperties;
   previewIframeSrc: SafeResourceUrl;
   editMode = false;
+  updatingPlayer = false;
```

Comment:
I'm a bit confused by the name of this variable.  Does it imply that the user can only update the player if not playing the latest episode?  Something like `playLatest` and `togglePlayLatest()` might be clearer.